src/Components/Resource/BadgesList.tsx

```diff
@@ -1,63 +1,25 @@
-import { useState, useEffect } from "react";
-import { getAnyFacility } from "../../Redux/actions";
-import { useDispatch } from "react-redux";
 import { SHIFTING_FILTER_ORDER } from "../../Common/constants";
+import routes from "../../Redux/api";
+import useQuery from "../../Utils/request/useQuery";
+
+function useQueryHook(facilityId: string | undefined) {
```

Comment:
Use proper names that identify what it does. Naming a function `useQueryHook` does not indicate what it queries although it queries a facility.

```suggestion
function useFacilityQuery(facilityId: string | undefined) {
```